content/guide/github.md

```diff
@@ -56,20 +56,11 @@ With a project on GitHub, it’s possible for others to make changes and commits
 - You can make them a collaborator on your repository in which case they can work on it directly just as you would. This is generally easier to manage, but grants them full access to your repository.
 - If your project is hosted in a GitHub Organization account, you can add them to the organization team.
 
-When collaborating with others, be communicative about what areas of the project you’re working on and try to avoid cross overs.
-
-Use branches as a way of managing
-
-
-There are a number of guides for forking repositories and making pull requests.
-
-When working with others on a single project,
-
-If you have multiple people connected to a project and making changes and pushing commits to the GitHub repository, there’s a good chance you will at some point make changes that are incompatible with someone else’s changes. Git is designed to deal with these sorts of issues.
-
-The trick with multiple people working in a single project is managing potential change conflicts. If two make people make different changes to a single file and then try to commit those changes to the central copy of the project on GitHUb
+When collaborating with others, be communicative about what areas of the project you’re working on and try to avoid cross overs. If you have multiple people connected to a project and pushing commits to the GitHub repository, there’s a good chance you will at some point make changes that are incompatible with someone else’s changes. Git is designed to deal with these sorts of issues.
 
+The trick with multiple people working in a single project is managing potential change conflicts. Issues may occur if two make people make different changes to a single file and then try to commit those changes to the central copy of the project on GitHub.
```

Comment:
"If you have multiple people connected to a project and pushing commits to the GitHub repository, there’s a good chance you will at some point make changes that are incompatible with someone else’s changes." seems repetitive with "there’s a good chance you will at some point make changes that are incompatible with someone else’s changes" in the paragraph above. Consider combining the paragraphs?